.vscode/launch.json

```diff
@@ -2,31 +2,32 @@
   // Use IntelliSense to learn about possible attributes.
   // Hover to view descriptions of existing attributes.
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
+  "version": "0.3.0",
   "configurations": [
     {
       "name": "Run Tests",
       "request": "launch",
       "runtimeArgs": ["run", "test"],
       "runtimeExecutable": "npm",
+      "cwd": "${workspaceFolder}",
```

Comment:
I needed to add `cwd` in order to run the debugger with tests (was not referencing correct tests). Let me know if this wasn't necessary.